src/applications/gi/constants.js

```diff
@@ -1,2 +1,6 @@
 // WAIT_INTERVAL is in milliseconds.
 export const WAIT_INTERVAL = 333;
+
+export const ARIA_LABELS = Object.freeze({
```

Comment:
Can we name it ariaLabels?

Reply:
I would like to change waitInterval too, but not necessary for this change.

Reply:
Sure thing.